controllers/grid/settings/sections/SectionGridHandler.inc.php

```diff
@@ -202,19 +202,32 @@ function updateSection($args, $request) {
 	 */
 	function deleteSection($args, $request) {
 		$journal = $request->getJournal();
-
+		
```

Comment:
It looks like you introduced some extra white-space. You should be able to get rid of this automatically on save with a setting in your code editor.

Reply:
Are you referring to the tabs in front of the line? I could not figure out how to remove those automatically in Notepad++, but it could be that I do not know the right word here. Notepad adds those automatically if you press enter and have been on an inteded line.